app/views/static_pages/success_stories.html.erb

```diff
@@ -6,14 +6,32 @@
   <div class="max-w-5xl mx-auto">
 
     <% @success_stories.each do |story| %>
-      <%= render CardComponent.new(classes: 'mb-10') do |card| %>
+      <%= render CardComponent.new(classes: 'mb-10', id: dom_id(story)) do |card| %>
         <% card.with_header do %>
           <div class="flex flex-col sm:flex-row items-center space-y-2 sm:space-y-0 sm:space-x-4">
             <div class="w-20 h-20 rounded-full overflow-hidden">
               <%= image_tag "success_stories/#{story.avatar_path_name}", alt: '', class: 'w-full shadow-inner aspect-auto' %>
             </div>
             <div class="flex flex-col text-center sm:text-left">
-              <p class="font-bold"><%= link_to story.student_name, story.social_media_link, class: 'underline hover:no-underline text-gray-600 hover:text-gray-800 dark:text-gray-200 dark:hover:text-gray-300' %></p>
+              <div class="flex flex-row gap-2">
```

Comment:
Related the previous suggestion - If we have a smaller icon, it'll look best of centered with the name.

```suggestion
              <div class="flex flex-row gap-2 items-center">
```